src/base.js

```diff
@@ -342,6 +342,14 @@ VRDisplay.prototype.fireVRDisplayPresentChange_ = function() {
   window.dispatchEvent(event);
 };
 
+VRDisplay.prototype.fireVRDisplayConnect_ = function() {
+  // Important: unfortunately we cannot have full spec compliance here.
+  // CustomEvent custom fields all go under e.detail (so the VRDisplay ends up
+  // being e.detail.display, instead of e.display as per WebVR spec).
+  var event = new CustomEvent('vrdisplayconnect', {detail: {display: this}});
```

Comment:
also another note: (FYI: this probably isn't an issue because the Polyfill realistically probably doesn't need to support IE 11.)

Edge is fine, but in IE 11 `CustomEvent` isn't exposed as a constructor (see [MDN](https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent/CustomEvent#Polyfill)). so you have to use `document.createElement` and `initCustomEvent`, like so:

```js
var event = document.createEvent('CustomEvent');
event.initCustomEvent(…);
```

same as [here](https://github.com/googlevr/webvr-polyfill/pull/244/files#diff-d7e697eff3913f1acaacac8002c3b05eR341) and probably elsewhere too.